crates/keystore/src/keystore/config.rs

```diff
@@ -42,6 +42,29 @@ pub struct KeystoreConfig {
         feature = "ledger-node"
     ))]
     pub(crate) remote_configs: Vec<crate::remote::RemoteConfig>,
+
+    #[cfg(feature = "substrate-keystore")]
+    pub(crate) substrate: Option<blueprint_std::sync::Arc<sc_keystore::LocalKeystore>>,
```

Comment:
```suggestion
    pub(crate) substrate: Option<sc_keystore::LocalKeystore>,
```

Does this need to be wrapped in an `Arc`?

Reply:
Yup, it does not implement clone, but mostly in Substrate when you work with keystore, you will be working with an Arc.

Reply:
Ah, okay LGTM.